CHANGELOG.md

```diff
@@ -18,6 +18,13 @@ please see [changelog_updates.md](docs/dev/changelog_updates.md).
 #### Patch Changes
 - Logginghouse-Client: Update logging-house-client extension to v1.1.0
 
+#### Patch Changes
+
+- API Wrapper:
```

Comment:
Guess the merge got messed up since the changelog now includes 2 times Patch Changes and the API-Wrapper single-contract agreement thingy is once under Minor Changes and once under Patch Changes now